### ISSUE TITLE
Lack of Observable gives Error in Flow Generator Plugin

### ISSUE THREAD

**Issue by egekorkan** [eclipse-thingweb/node-red#30]
Taking a TD with a property without observable, reports a JSON parse error at TD insertion (see screenshot below):

![image](https://github.com/user-attachments/assets/d383b360-5565-4955-8fc6-3a2d43324244)

Tracing it to the function:

> error:  SyntaxError: JSON.parse: unexpected character at line 81 column 20 of the JSON data
    replaceParamsAndIds http://localhost:1880/resources/@thingweb/node-red-node-wot/node-wot-plugin-lib.js:154
    makePropertyFlow http://localhost:1880/resources/@thingweb/node-red-node-wot/node-wot-plugin-lib.js:169
    createClientFlowUsingDashboard http://localhost:1880/resources/@thingweb/node-red-node-wot/node-wot-plugin-lib.js:83

Example TD:

```json
{
    "@context": "https://www.w3.org/2022/wot/td/v1.1",
    "@type": "Thing",
    "title": "Smart Clock",
    "securityDefinitions": {
        "nosec_sc": {
            "scheme": "nosec"
        }
    },
    "security": [
        "nosec_sc"
    ],
    "properties": {
        "time": {
            "readOnly": true,
            "type": "object",
            "properties": {
                "minute": {
                    "type": "integer",
                    "minimum": 0,
                    "maximum": 59
                },
                "hour": {
                    "type": "integer",
                    "minimum": 0,
                    "maximum": 23
                }
            },
            "forms": [
                {
                    "href": "coap://localhost:5686/smart-clock/properties/time",
                    "contentType": "application/json",
                    "op": [
                        "observeproperty",
                        "unobserveproperty"
                    ],
                    "subprotocol": "cov:observe"
                }
            ],
            "writeOnly": false
        }
    },
    "id": "urn:uuid:ac327610-31f5-4083-abf1-3730dbb59b1f",
    "description": "a smart clock that runs 60 times faster than real time, where 1 hour happens in 1 minute.",
}
```

@hidetak can you have a look?

**Comment by hidetak** [comment]
@egekorkan 
Thank you for pointing this out.
I have made the following corrections to the PR, along with other problems I have found.
https://github.com/eclipse-thingweb/node-red/pull/31

I would be happy to confirm this.

**Comment by egekorkan** [comment]
Fixed in #31
